src/Voice.tsx

```diff
@@ -857,7 +857,7 @@ export namespace Voice {
     /**
      * Custom parameters to send to the TwiML Application.
      */
-    params?: Record<string, any>;
+    params?: Record<string, string>;
```

Comment:
No unit tests related to this change?

Reply:
Not sure if we need unit tests for this since there's no logic actually backing this. Either the TS will compile or fail, on our end or a customer's end. WDYT?

Reply:
I think we need some kind of testing. If we accidentally change this API, how do we catch it during build process? Do we have a step in our build process that tests the types?